packages/smart-contracts/test/contracts/BatchConversionPayments.test.ts

```diff
@@ -386,56 +382,55 @@ describe('contract: BatchConversionPayments', async () => {
           },
         ],
         feeAddress,
-        { value: 1000 + 1 + 42 },
+        { value: 1000 + 1 + 42 }, // +42 in excess
       );
 
-      await checkETHBalances(
+      await expectETHBalanceDiffs(
         BigNumber.from(1000),
         BigNumber.from(1),
         BATCH_FEE,
-        beforeETHBalanceFrom,
-        beforeETHBalanceTo,
-        beforeETHBalanceFee,
+        initialFromETHBalance,
+        initialToETHBalance,
+        initialFeeETHBalance,
       );
     });
 
     it('make 1 ETH payment with 1-step conversion', async function () {
```

Comment:
```suggestion
    it('make 1 ETH payment with 1-step conversion', async () => {
```
This is the formalism we use generally